mod/01_filter/set/abstract/filter.rb

```diff
@@ -315,8 +296,8 @@ def checkbox_formgroup title, options, default=nil
   end
 
   view :importance_formgroup do
-    options = ["Not Voted", "Upvoted", "Downvoted"]
-    checkbox_formgroup "My Vote", options, ["upvoted", "not voted"]
+    options = ["I voted FOR", "I voted AGAINST", "I did NOT vote"]
+    checkbox_formgroup "My Vote", options, ["i voted for", "i did not vote"]
```

Comment:
fwiw, it's ok if the hidden names are more technical.  in fact, they may be easier for developers to understand.  but not important to change here.